docs/custom-android-back-button-handling.md

```diff
@@ -12,6 +12,7 @@ As an example, consider a screen where user is selecting items in a list, and a
 
 Returning `true` from `onBackButtonPressAndroid` denotes that we have handled the event, and react-navigation's listener will not get called, thus not popping the screen. Returning `false` will cause the event to bubble up and react-navigation's listener will pop the screen.
 
+#TODO
```

Comment:
This will be much easier now with the `useFocusEffect` hook:

```js
useFocusEffect(() => {
  const onBackPress = () => {
    // Do something
  };

  BackHandler.addEventListener('hardwareBackPress', onBackPress);

  return () => BackHandler.removeEventListener('hardwareBackPress', onBackPress);
}, []);
```